src/lib/markbind/src/lib/nunjucks-extensions/nunjucks-date.js

```diff
@@ -0,0 +1,19 @@
+const Moment = require('moment');
+
+const DEFAULT_FORMAT = 'ddd D MMM'; // e.g. Thu 27 Aug
+
+/**
+ * Calculates the result of adding {days} days to the base date
+ * @param base
+ * @param days
+ */
+function calculateTarget(base, days) {
+  return Moment(base).add(days, 'days');
+}
+
+function filter(baseDate, format = DEFAULT_FORMAT, day = 0) {
```

Comment:
Is it possible to allow the user to set a `DEFAULT_FORMAT`? Maybe somewhere in site.json.

Then we can change the arguments to `filter` to `filter(baseDate, day = 0, format = DEFAULT_FORMAT)`.

This is assuming that the most frequent use case would be the author having only 1 format they want to use, so being able to do `{{ base | date(7*2 + 1) }}` and avoid typing the format all the time might be easier syntax.

Reply:
Fair point, I suppose if there really was a need we could also split the filter into 2 separate filters in a future PR, like `{{ baseDate | addDays(7 * 2 + 1) | formatDate(format) }}`.

Configuring a custom `DEFAULT_FORMAT` isn't too hard but I'm not sure if it will be very useful. I don't think that it matters enough either way, since you could just pass in an extra argument. An alternative is to define a nunjucks macro (`customFormat` defined in `variables.md`)
```
{% macro customDate(baseDate, days) %}
{{ baseDate | date(customFormat, days) }}
{% endmacro %}

Usage: {{ customDate(baseDate, 5) }}
```

Reply:
Could we still allow this, but without the configuration? :o 
( swapping the `day` and `format` in `format = DEFAULT_FORMAT, day = 0) {` )